content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md

```diff
@@ -0,0 +1,481 @@
+
+- Analyze the surrounding sounds with the onboard microphone of the Nicla Voice and run artificial intelligence algorithms at the edge.
+- Correctly identify if a door has been opened in an ordinary manner or suffered a forced attempt.
+- Report opening and intruder detection events through Bluetooth® Low Energy to a Host.
+- The host must forward the data from the Nicla Voice to the Arduino IoT cloud using an Internet connection (Wi-Fi® in this case).
+- Create an Arduino Cloud dashboard that syncs in real time to inform and alert the user.
+- Gather real-world data to train a Machine Learning model.
+
+## Hardware and Software Requirements
+
+![Nicla Voice, Portenta H7, and 3D enclosures](assets/hardware.png)
+
+### Hardware Requirements
+- Nicla Voice
+- 3.7 V LiPo battery
+- Portenta H7
+- USB-C® cable
+- Micro USB cable
+- 5 VDC wall power adapter
+- DC brushless fan Model: LD3007MS
+- [Nicla Voice and Portenta H7 3D Printed enclosures](assets/Intruder_Detector_3D_Files.zip)
+
+### Software Requirements
+- [Arduino IDE 1.8.10+](https://www.arduino.cc/en/software), [Arduino IDE 2.0+](https://www.arduino.cc/en/software), or [Arduino Web Editor](https://create.arduino.cc/editor)
+- If you are going to use an offline Arduino IDE, you must install the following libraries: `ArduinoBLE`, `ArduinoIoTCloud`, and `Arduino_ConnectionsHandler`. You can install them using the Arduino IDE Library Manager.
+- The [intruder detector example code](assets/Nicla_Portenta_Codes.zip)
+- [Edge Impulse® CLI](https://docs.edgeimpulse.com/docs/edge-impulse-cli/cli-overview), which you can install following these [instructions](https://docs.edgeimpulse.com/docs/edge-impulse-cli/cli-installation)
+- Nicla Voice audio [firmware](https://cdn.edgeimpulse.com/firmware/arduino-nicla-voice-firmware.zip) to upload recordings to Edge Impulse®.
+- [Syntiant® uploader and precompiled ML model](assets/Syntiant_Uploader.zip)
+- [Arduino Agent](https://create.arduino.cc/getting-started/plugin/welcome) to add the Portenta H7 to the IoT Cloud
+
+## The Machine Learning Model
+
+The Machine Learning is a way of programming in which we make our device process raw data from sensors and learn based on extracted meaningful features that will be later used for identifying or predicting the originally trained scenarios.
+
+To be able to identify when the door is being opened or when it is being forced by someone, we trained a machine learning model using the [Edge Impulse](https://www.edgeimpulse.com/) integration with the [Arduino Cloud](https://cloud.arduino.cc/), that will be running repetitively in the Nicla Voice.
+
+First, we flashed the Nicla Voice with an [audio ingestion firmware](https://cdn.edgeimpulse.com/firmware/arduino-nicla-voice-firmware.zip) to record and upload samples to our Edge Impulse project of three different scenarios that we want to identify later. 
```

Comment:
Machine Learning Tools project instead of Edge Impulse

Reply:
Changed to:

First, we flashed the Nicla Voice with an [audio ingestion firmware](https://cdn.edgeimpulse.com/firmware/arduino-nicla-voice-firmware.zip) to record and upload samples to our Machine Learning Tools project of three different scenarios that we want to identify later.